### DISCUSSION TITLE
UTF-8?

### DISCUSSION THREAD

**Discussion by SLothinatorSYS** [MahApps/MahApps.Metro#4438] [Q&A]
Hi!

I am using Windows10UpgradeGUI that uses MahApps.Metro. I am just wondering, MahApps.Metro supports Swedish letters as in ÅÄÖ? cause when launching Windows10UpgradeGUI it looks like the picture I posted. Maybe MahApps.Metro supports the Swedish letters but not Windows10UpgradeGUI? Been trying to resolve this issue and can't seem to fix it so came here to see if anyone knows how to fix this. I know Windows10UpgradeGUI is pretty old but it does exactly what I want it to. It's using MahApps.Metro 1.5.0 and MahApps.Metro.Resources 0.6.1.0

![image](https://github.com/MahApps/MahApps.Metro/assets/151180177/dcee5112-604a-4996-825a-e69265e5111c)

Thanks in advance!

**Comment by timunie** [comment]
Guess it depends on the Font defined. You may need to specify another font?

**Reply by SLothinatorSYS** [reply]
The font is specified in a .dll file. I did manage to open it and edit it but can't repackage into a new .dll file

![image](https://github.com/MahApps/MahApps.Metro/assets/151180177/2b85fac8-e3a9-4941-b26f-1cd6e6703bd5)

**Reply by punker76** [reply]
Maybe it's also better to use the latest MahApps version instead using v1.5.0 and MahApps.Metro.Resources v0.6.1.0, these ones are outdated and I will not look in to it anymore.
